README.md

```diff
@@ -93,23 +91,32 @@ Install this project with:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/234849612-9b7a54dc-b166-4c50-b9e3-2f3c52d09c16.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalized 👍 because some of your commit messages are not capitalized 😨 Please consider this issue to make your pull request professional and descriptive ☺️

Reply:
sure we will try to write meaningful and capitalized  next time